control-plane/pkg/core/config/utils.go

```diff
@@ -136,3 +136,41 @@ func DurationMillisFromISO8601String(durationStr *string, defaultDurationMillis
 func IncrementContractGeneration(ct *contract.Contract) {
 	ct.Generation = (ct.Generation + 1) % (math.MaxUint64 - 1)
 }
+
+// MergeEgressConfig merges the 2 given egress configs into one egress config prioritizing e0 values.
+func MergeEgressConfig(e0, e1 *contract.EgressConfig) *contract.EgressConfig {
+	if e0 == nil {
+		return e1
```

Comment:
what if `e1` is nil? :smile:

Reply:
That is fine.

Reply:
In that case we have `MergeEgressConfig(nil, nil) = nil`